Students/Stoica S. Razvan Cosmin/Assignment1/assign3/src/assign3/assign3.java

```diff
@@ -0,0 +1,38 @@
+package assign3;
+
+public class assign3
+
+{
+	public static int checkP(int n) {
+		int aux = n, inv = 0;
+		while (aux != 0) {
+			inv = inv * 10 + aux % 10;
+			aux = aux / 10;
+		}
+		if (n == inv)
+			return 1;
+		else
+			return 0;
+	}
+
+	public static void Palindrom() {
```

Comment:
According to [this](http://www.oracle.com/technetwork/java/codeconventions-135099.html) the first letter of a method name should be lowercase.